include/constants/battle.h

```diff
@@ -342,7 +343,8 @@
 #define MOVE_EFFECT_THROAT_CHOP         0x43
 #define MOVE_EFFECT_INCINERATE          0x44
 #define MOVE_EFFECT_BUG_BITE            0x45
-#define NUM_MOVE_EFFECTS                0x46
+#define MOVE_EFFECT_SKY_DROP			0x46
```

Comment:
The value of the `MOVE_EFFECT_SKY_DROP` constant is not properly aligned with the values of the constants above and below it.

Reply:
Also, you're using tabs when you should be using empty spaces.